src/app-layout/visual-refresh/navigation.tsx

```diff
@@ -48,7 +47,7 @@ export default function Navigation() {
     }
   };
 
-  const isUnfocusable = hasDrawerViewportOverlay && isToolsOpen && !toolsHide;
+  const isUnfocusable = hasDrawerViewportOverlay && (!isNavigationOpen || isToolsOpen);
```

Comment:
Do we still need to check `toolsHide`? In case when `toolsOpen={true} toolsHide={true}`. Tried with this setting, the navigation will disappear while tools is not hidden, only background colour remains.

Reply:
Good point. Updated the code, added a test